src/main/java/uk/gov/hmcts/cmc/ccd/client/SubmitForCaseworkerApi.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.cmc.ccd.client;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.hmcts.cmc.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.cmc.ccd.client.model.CaseDetails;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@FeignClient(name = "core-case-data-api", url = "${core_case_data.api.url}")
+public interface SubmitForCaseworkerApi {
```

Comment:
not sure if we want multiple classes for caseworker and citizen,
are the calls quite different?

thoughts?

Reply:
They have different endpoints for both. Each require different role etc. Not very sure about the other differences.

Reply:
I suppose having them separate does not hurt too much? I guess with Feign we cannot share anything anyway, it's just call definitions?

Reply:
May be combine both `start` and `submit` in one interface definition for `caseworker`.

Reply:
we could you same client for both citizen and caseworkers, they can be put in path param

Reply:
But that would be our assumption that might change later at CCD side.

Reply:
I'm happy with it how it is